README.md

```diff
@@ -16,8 +16,9 @@ It uses [Turborepo](https://turborepo.org/) and contains:
   └─ Recommended extensions and settings for VSCode users
 apps
   ├─ expo
-  |   ├─ Expo SDK 46
+    | ├─ Expo SDK 47
```

Comment:
```suggestion
  | ├─ Expo SDK 47
```

Reply:
fix it, sorry for merging without checking it